src/models/Order.ts

```diff
@@ -0,0 +1,57 @@
+import {Types, Schema, model} from 'mongoose'; 
+
+const OrderSchema = new Schema({
+    id: {
+        type: Types.ObjectId, 
+        unique: true,
+        required: true, 
+    }, 
+    partner: {
+        type: Types.ObjectId,
+        ref: "Partner", 
+        required: true,
+    }, 
+    datePlaced: {
+        type: Date,
+        default: () => Date.now(), 
+        required: true,
+    }, 
+    dateCompleted: {
+        type: Date,
+        default: null,  
+    }, 
+    status: {
+        type: String, 
+        note: "enum with options: PLACED, OPEN, FILLED, CANCELLED",
```

Comment:
Please get rid of the note field and replace it with an enum field:
enum: ['PLACED', 'OPEN', 'FILLED', 'CANCELLED']